### PR TITLE
refactor: update test method names 

### PR DESCRIPTION
**Pull request by yusufaine**
Changes were made with respect to the discussion [here](https://github.com/AY2122S2-CS2103T-T17-2/tp/pull/105#issuecomment-1072001031).

### CODE REVIEW

**Review by gigituang — COMMENTED** [review]
LGTM! i will update the ListCommandTest according

**Review by gigituang — APPROVED** [review]

**Review by blaketingyu — APPROVED** [review]
LGTM! 🌮 I believe you've covered most of the test cases already eg validity, equality.

File: src/test/java/seedu/address/model/recipe/CompletionTimeTest.java [hunk 1]
```diff
@@ -34,14 +34,19 @@ public void toString_test() {
     }
 
     @Test
-    public void equalityCheck() {
+    public void equals_test() {
```
Comment (blaketingyu) [inline src/test/java/seedu/address/model/recipe/CompletionTimeTest.java:37]: Just a reminder, if we are standardizing our test method names to be more specific/detailed. Then we should change the equals_test method name.

```suggestion
    public void test_equalityCheckForValidAndInvalidCompletionTime() {
```

File: src/test/java/seedu/address/model/recipe/CompletionTimeTest.java [hunk 2]
```diff
@@ -34,14 +34,19 @@ public void toString_test() {
...
 
         // Test CompletionTime::equals
         assertNotEquals(c1, c1.value);
         assertEquals(c1, c1);
         assertEquals(c1, c1Copy);
         assertNotEquals(c1, c2);
+    }
+
+    @Test
+    public void hashCode_test() {
```
Comment (blaketingyu) [inline src/test/java/seedu/address/model/recipe/CompletionTimeTest.java:48]: Maybe this one could be something like
```suggestion
    public void test_equalityCheckForHashCodeOfTheSameObject() {
```

**Review by blaketingyu — CHANGES_REQUESTED** [review]
Maybe the naming convention could be more specific/detailed as what @aidanaden mentioned. 
The method names I've suggested could be improved as well. It's just a suggestion 🌮 😄

**Review by aidanaden — APPROVED** [review]
LGTM! I don't think equals_test and hashcode_test requires super specific naming conventions considering it's a common test done on all classes.

🔥 🔥 🔥

**Pull request merged**